plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -140,43 +132,144 @@ class ServiceDestroyModal extends React.Component {
     }, REDIRECT_DELAY);
   }
 
-  render() {
-    const { isPending, onClose, open, service } = this.props;
+  getCloseButton() {
+    const { onClose } = this.props;
+
+    return (
+      <div className="row text-align-center">
+        <button
+          className="button button-primary button-medium"
+          onClick={onClose}
+        >
+          Close
+        </button>
+      </div>
+    );
+  }
+
+  getDestroyFrameworkModal() {
+    const { open, service, onClose, intl } = this.props;
+    const serviceName = service ? service.getId() : "";
+
+    return (
+      <Modal
+        header={this.getModalHeading()}
+        footer={this.getCloseButton()}
+        modalClass="modal"
+        onClose={onClose}
+        open={open}
+        showHeader={true}
+        showFooter={true}
+        subHeader={this.getSubHeader()}
+      >
+        <p>
+          {intl.formatMessage({
+            id: "SERVICE_ACTIONS.DELETE_SERVICE_FRAMEWORK"
+          })}
+          <a
+            href="https://docs.mesosphere.com/service-docs/"
+            target="_blank"
+            title={intl.formatMessage({
+              id: "COMMON.DOCUMENTATION_TITLE"
+            })}
+          >
+            {intl.formatMessage({
+              id: "COMMON.DOCUMENTATION"
+            })}
+          </a>
+          {intl.formatMessage({
+            id: "SERVICE_ACTIONS.DELETE_SERVICE_FRAMEWORK_2"
+          })}
+        </p>
+        <div className="flush-top snippet-wrapper">
+          <ClickToSelect>
+            <pre className="prettyprint flush-bottom">
+              dcos packages uninstall --app-id={serviceName}
+            </pre>
+          </ClickToSelect>
+        </div>
+        {this.getErrorMessage()}
+      </Modal>
+    );
+  }
 
-    let itemText = "Service";
+  getDestroyServiceModal() {
+    const { onClose, open, service, intl } = this.props;
+    const serviceName = service ? service.getId() : "";
+    const isButtonDisabled = this.state.isButtonDisabled;
+
+    let itemText = `${StringUtil.capitalize(UserActions.DELETE)}`;
```

Comment:
How do we translate the `itemText`? Feels wrong to add intl support for everything but this one.